Added entry to describe the getFullYear method in javascript (#2286)
Codecademy/docs: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md

---
Title: 'getFullYear()' 

> Description: 'Return the current full year' 

Reviewer: ```suggestion
Description: 'Return the current full year.' 
```
This is wrong, It gets the four digit year of any given date.

Subjects: 
  - 'Web Development'
Tags: 
  - 'Date'
  - 'Method'
CatalogContent: 
  - 'paths/web-development'
---

> The **getFullYear** method returns the current four digit year

Reviewer: ```suggestion
The **getFullYear** method returns the current four digit year.
```
This is wrong, It gets the four digit year of any given date.

## Syntax

```js
myDate.getFullYear()

> ```

Reviewer: Needs a sentence describing the syntax, basically what `myDate` represents.

## Example

> 

Reviewer: Needs a sentence explaining what the example is doing.

Author: I have added this

```js

> const getYear = new Date().getFullYear()

Reviewer: ```suggestion
const getYear = new Date().getFullYear()
```
Change the variable name so it isn't the name of another JavaScript date method.

Author: Good point I have changed this

> // returns 2023

Reviewer: ```suggestion
```
This entry will remain up beyond the current year, so remove this comment. Also, add a line that does something with the result, such as print it to the console.

---
Title: 'getFullYear()' 

> Description: 'Returns the four digit year for the given date' 

Reviewer 2: ```suggestion
Description: 'Returns the four digit year for the given date.' 
```

Subjects: 
  - 'Web Development'
Tags: 
  - 'Date'

>   - 'Method'

Reviewer 2: ```suggestion
  - 'Methods'
```

CatalogContent: 

>   - 'paths/web-development'

Reviewer 2: ```suggestion
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
```

---

> The **getFullYear** method returns a four digit year for the date according to local time.

Reviewer 2: ```suggestion
The **`getFullYear()`** method returns a four digit year for the date according to local time.
```

## Syntax

```js
myDate.getFullYear()
```

> The returned value of `.getFullYear()` method is an integer for the provided date.

Reviewer: ```suggestion
The returned value of `.getFullYear()` method is an integer representing the year for the provided date, `myDate`.
```

> The returned value of `.getFullYear()` method is an integer representing the year for the provided date `myDate`

Reviewer 2: ```suggestion
The returned value of `.getFullYear()` method is an integer representing the year for the provided date `myDate`.
```

## Example

> In the example below the `year` variable store the value of a new Date object and then logs the year to the console.

Reviewer: ```suggestion
In the example below the `year` variable stores the year returned from of a new Date object and then logs it to the console.
```
